production/helm/loki/templates/_helpers.tpl

```diff
@@ -198,11 +198,12 @@ Docker image name
 Generated storage config for loki common config
 */}}
 {{- define "loki.commonStorageConfig" -}}
-{{- $bucketName := required "Please define loki.storage.bucketNames.chunks" (dig "storage" "bucketNames" "chunks" "" .Values.loki) }}
 {{- if .Values.loki.storage.use_thanos_objstore -}}
+{{- $bucketName := required "Please define loki.storage.bucketNames.chunks" (dig "storage" "bucketNames" "chunks" "" .Values.loki) }}
 object_store:
   {{- include "loki.thanosStorageConfig" (dict "ctx" . "bucketName" $bucketName) | nindent 2 }}
 {{- else if .Values.minio.enabled -}}
+{{- $bucketName := required "Please define loki.storage.bucketNames.chunks" (dig "storage" "bucketNames" "chunks" "" .Values.loki) }}
```

Comment:
@jkroepke I don't think this values path needs to be defined when minio is being used, as for instannce it is not defined in the example override values here: https://github.com/grafana/loki/blob/main/production/helm/loki/simple-scalable-values.yaml